packages/calcite-components/src/components/input-message/input-message.scss

```
@@ -3,34 +3,39 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-input-message-spacing-value: The top margin spacing above the component.
+ * @prop --calcite-input-message-spacing-value: [Deprecated] Use `--calcite-input-message-spacing`.The top margin spacing above the component.
```

```suggestion
 * @prop --calcite-input-message-spacing-value: [Deprecated] Use `--calcite-input-message-spacing`. The top margin spacing above the component.
```

Idk if we usually specify the direction of the spacing in the token names so I have two suggestions for description change:
1.  Specifies the amount of top margin above the component.
2. Specifies the margin spacing at the top of the component.